package.json

```diff
@@ -10,14 +10,19 @@
         "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
     },
     "devDependencies": {
-        "@babel/preset-react": "^7.8.3",
+        "@babel/preset-react": "^7.0.0",
         "@types/node": "^12.12.28",
         "@types/react": "^16.9.22",
         "@types/react-dom": "^16.9.5",
         "axios": "^0.19",
+        "bootstrap": "^4.0.0",
         "cross-env": "^7.0.1",
+        "jquery": "^3.2",
         "laravel-mix": "^5.0.1",
         "lodash": "^4.17.13",
+        "popper.js": "^1.12",
+        "react": "^16.2.0",
+        "react-dom": "^16.2.0",
```

Comment:
I don't think we should need react and react-dom here in the devDependencies because we've already got them in the dependencies